deployment/load_production_backup.sh

```diff
@@ -47,6 +47,7 @@ sudo -H -u $USERNAME $ENVDIR/bin/pip install -r requirements.txt
 
 # compilemessages and compress regularly fail without any real issue.
 sudo -H -u $USERNAME $ENVDIR/bin/python manage.py compilemessages || true
+sudo -H -u $USERNAME $ENVDIR/bin/python manage.py scss --release
```

Comment:
that doesn't really fit into loading backups. as you already added it to update_production.sh, I'd remove it here.

Reply:
Afaik, `collectstatic --noinput` had the same intent – to update all static assets, including sass files. And I can’t quite see where we are calling `update_production.sh` after `load_production_backup.sh`, so I’m not sure where I would put this call if not here (without changing the script structure).

Reply:
yeah, on a second look it seems like load_production_backup is meant to do a full rollback and not just load the backup.